fedimint-cli/src/main.rs

```diff
@@ -1,5 +1,12 @@
 use fedimint_cli::FedimintCli;
 use fedimint_core::fedimint_build_code_version_env;
+#[cfg(not(target_env = "msvc"))]
+use jemallocator::Jemalloc;
+
+#[cfg(not(target_env = "msvc"))]
+#[global_allocator]
+// rocksdb suffers from memory fragmentation when using standard allocator
+static GLOBAL: Jemalloc = Jemalloc;
```

Comment:
do even care windows? (msvc)

Reply:
if yes, probably the dep has to be gated something like so:

```
[target.'cfg(not(target_env = "msvc"))'.dependencies]
jemallocator = "0.5"
```

Reply:
We probably don't, we don't have any CI to verify anything about it. I just copied the boilerplate from jemalloc's README and I don't think it hurts.

Reply:
Will do it in https://github.com/fedimint/fedimint/pull/6357